src/main/markdown/examples.md

```diff
@@ -207,6 +191,19 @@ was built using GWT. Use the free trial to see it in action.</div>
 
 <div class="project">
   <div class="screenshot">
+    <a href="https://getbookedin.com/?cid=256" target="_blank"><img src="images/bookedin.png" alt="BookedIN"/></a>
+  </div>
+  <div class="info">
+    <div class="name"><a href="https://getbookedin.com/?cid=256" target="_blank">BookedIN</a></div>
+    <div class="snippet">BookedIN is an online appointment booking, scheduling and payment system for small business.
+      Customers can book and pay for appointments online, and the business owner can automate their internal booking
+      process, reducing their administrative workload. BookedIN uses <a href="https://gwtp.arcbees.com">GWTP</a>
+      which has helped to simplify and accelerate development. </div>
```

Comment:
I would remove this line. We don't want to advertise any third-party framework in the gwt website. You can mention which technology is used but that's it